src/site/markdown/server/manage-webadmin.md

```diff
@@ -358,7 +358,45 @@ Response codes:
 
 The kind of task scheduled depends on the action parameter. See below for details.
 
-### Recomputing JMAP fast message view projection
+#### Fixing mailboxes inconsistencies
+
+This task is only available on top of Guice Cassandra products.
+
+```
+curl -XPOST /mailboxes?task=solveInconsistencies
+```
+
+Will schedule a task for fixing the mailbox projection.
```

Comment:
what is "mailbox projection"?

Reply:
s/mailbox projection/mailbox deduplicated object/ ?